Mage.Sets/src/mage/cards/k/KjeldoranGuard.java

```diff
@@ -0,0 +1,140 @@
+package mage.cards.k;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.DelayedTriggeredAbility;
+import mage.abilities.condition.CompoundCondition;
+import mage.abilities.condition.InvertCondition;
+import mage.abilities.condition.common.IsPhaseCondition;
+import mage.abilities.condition.common.OpponentControlsPermanentCondition;
+import mage.abilities.costs.common.TapSourceCost;
+import mage.abilities.decorator.ConditionalActivatedAbility;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.SacrificeTargetEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.common.FilterLandPermanent;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetControlledCreaturePermanent;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Alex-Vasile
+ */
+public class KjeldoranGuard extends CardImpl {
+
+    private static final FilterLandPermanent snowLandFiler = new FilterLandPermanent("a snow land");
+
+    static { snowLandFiler.add(SuperType.SNOW.getPredicate()); }
+
+    public KjeldoranGuard(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{W}");
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.SOLDIER);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(1);
+
+        // Target creature gets +1/+1 until end of turn.
+        // When that creature leaves the battlefield this turn, sacrifice Kjeldoran Guard.
+        // Activate only during combat and only if defending player controls no snow lands.
+        CompoundCondition snowLandAndCombatCondition = new CompoundCondition(
+                new InvertCondition(new OpponentControlsPermanentCondition(snowLandFiler)),
```

Comment:
This says defending player, and after some research this seems to refer to the player being attacked by the targeted creature. I've actually contacted the magic rules manager and lead editor about this and a few other cards with similarly ambiguous wordings.